src/backend/mod.rs

```diff
@@ -1,2 +1,26 @@
+//! Contains the different backends to get the screenshot from.
+
 pub mod wayland;
-pub mod xorg;
+pub mod x11;
+
+/// A general backend error enum.
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    /// Represents that an error occured while trying to get a screenshot on X11.
+    #[error(transparent)]
+    X11(#[from] x11::Error),
+}
+
+/// The main function of this module.
+///
+/// This function returns an rgb-image for each screen (or "monitor" in other
+/// words).
+pub fn get_images() -> Result<Vec<image::RgbImage>, Error> {
+    let is_running_wayland = std::env::var("WAYLAND_DISPLAY").is_ok();
```

Comment:
This env must not be set. It defaults then to wayland-0 as value.

Reply:
oh ok. I saw this solution from a stack-overflow post xD

Reply:
Fixed